app/destination/page.js

```diff
@@ -1,54 +1,74 @@
 "use client";
 
-import { useState } from 'react';
+import { useState } from "react";
 
-import styles from '@/components/destination/destination.module.css';
-import { AddWishlistItem } from '@/components/destination/AddWishlistItem';
-
-// TASK - React 1 week 2
-// Move this to its own file
-const PlanetWishlistItem = ({
-  name,
-  onRemove,
-  thumbnail,
-}) => {
-  return (
-    <div className={styles.wishlistItem}>
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
-    </div>
-  );
-}
+import styles from "@/components/destination/destination.module.css";
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
+import { PlanetWishlistItem } from "./PlanetWishListItem";
+import { Tulpen_One } from "next/font/google";
+import { PlanetCard } from "./PlanetCard";
 
+const Planets = [
```

Comment:
Small comment for readability:
This should be a lowercase letter:
`planets`

Uppercase varaibles should be only for React components or classes (or types in TypeScript) :)